website/docs/reference/warehouse-setups/trino-setup.md

```diff
@@ -150,26 +149,27 @@ trino:
 
 <TabItem value="kerberos">
 
-#### Kerberos Fields
-
-In addition to specifying  `method: kerberos`, the table below gives the kerberos-relevant parameters.
+The following table lists the authentication parameters to set for Kerberos.
 
-For addiontal information, refer to Trino's doc page on [Kerberos Authentication](https://trino.io/docs/current/security/kerberos.html)
+For more information, refer to [Kerberos Authentication](https://trino.io/docs/current/security/kerberos.html) in the Trino docs.
 
 | Profile field                               | Example             | Description                                                      |
 | ------------------------------------------- | ------------------- | ---------------------------------------------------------------- |
+| `method` | `kerberos`| Set Kerberos as the authentication method. |
 | `user`                                      | `commander`         | Username for authentication                                      |
 | `keytab`                                    | `/tmp/trino.keytab` | Path to keytab                                                   |
 | `krb5_config`                               | `/tmp/krb5.conf`    | Path to config                                                   |
 | `principal`                                 | `trino@EXAMPLE.COM` | Principal                                                        |
-| `service_name` (optional)                   | `abc123`            | Service name  (default is 'trino')                               |
+| `service_name` (optional)                   | `abc123`            | Service name (default is `trino`)                               |
 | `hostname_override` (optional)              | `EXAMPLE.COM`       | Kerberos hostname for a host whose DNS name doesn't match        |
 | `mutual_authentication` (optional)          | `false`             | Boolean flag for mutual authentication                           |
-| `force_preemptive` (optional)               | `false`             | Boolean flag for preemptively initiate the Kerberos GSS exchange |
+| `force_preemptive` (optional)               | `false`             | Boolean flag to preemptively initiate the Kerberos GSS exchange |
 | `sanitize_mutual_error_response` (optional) | `true`              | Boolean flag to strip content and headers from error responses   |
-| `delegate`  (optional)                      | `false`             | Boolean flag for credential delgation (`GSS_C_DELEG_FLAG`)       |
+| `delegate`  (optional)                      | `false`             | Boolean flag for credential delegation (`GSS_C_DELEG_FLAG`)       |
 
-#### Sample `profiles.yml` for Kerberos
+<br/>
+
+#### Example profiles.yml for Kerberos
```

Comment:
non-blocking nit: is this a style choice to not do code formatting within headers?

Reply:
yah, it's a style guide thing. having multiple "font decorations" can be distracting since the heading is by default bolded